bleak/exc.py

```diff
@@ -9,6 +9,14 @@ class BleakError(Exception):
     pass
 
 
+class BleakPermissionError(Exception):
+    """
+    Exception which is raised if the access to the bluetooth peripheral is denied.
+    """
+
+    pass
```

Comment:
I think this should have an additional argument that takes a new enum value similar to `CBManagerAuthorization` and have a property to access it. This way, users can programatically handle different kinds of errors without having to rely on string scraping.